app/components/participant-creator/component.js

```diff
@@ -32,13 +32,20 @@ export default Ember.Component.extend({
     _generate(batchSize, tag) {
         var ret = [];
         for (let i = 0; i < batchSize; i++) {
-            ret.push(`${makeId(5)}${tag ? `-${tag}` : ''}`);
+            var generate = true;
+            while (generate) {
+                var id = `${makeId(5)}${tag ? `-${tag}` : ''}`;
+                if (!ret.contains(id) || !this.get('store').peekRecord('account', id)) {
```

Comment:
When an account is created, it is pushed into the ember-data local store (records known to exist). However, people won't likely be batch creating all accounts at once.

If there are 5,000 accounts on the server previously, and we batch-create 10, we need a way to ask if those 10 proposed IDs previously exist. (without forcing the user to download every account in the database first and filter locally)